python/cuda_cccl/cuda/compute/_bindings_impl.pyx

```diff
@@ -586,6 +591,7 @@ cdef class Iterator:
     cdef object state_obj
     cdef object host_advance_obj
     cdef cccl_iterator_t iter_data
+    cdef TypeInfo value_type
```

Comment:
There is no need to add this class member, as it is duplicated in `self.iter_data.value_type`.

We can implement the property as follows without this field:

```python
    @property
    def value_type(self):
        cdef cccl_type_info _ti = self.iter_data.iter_data.value_type 
        return TypeInfo(_ti.size, _ti.alignment, _ti.type)
```